internal/policy/container/has_required_labels.go

```diff
@@ -24,15 +24,17 @@ func (p *HasRequiredLabelsCheck) Validate(ctx context.Context, imgRef certificat
 		return false, fmt.Errorf("could not retrieve image labels: %v", err)
 	}
 
-	return p.validate(labels)
+	return p.validate(ctx, labels)
```

Comment:
Would it make sense to reach into the struct for the context instead of accepting it in the internal validate function?

That would require passing a context into the check at init time instead of at Validate() time... Maybe that's worse.

Reply:
I'd rather pass at the point of use. I don't think it's good practice to store a context like that.